### DISCUSSION TITLE
Repos disappeared!!!!

### DISCUSSION THREAD

**Discussion by manage-strategizeai** [community/community#49317] [Repositories]
### Select Topic Area

Bug

### Body

Some repos just disappeared... I need help. They weren't deleted. They just don't exist anymore...

**Comment by Matteo-it** [comment]
Go on Your **Profile > Repositories** and here you have all the repo created!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @manage-strategizeai ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by MuhammadArslanKhan96** [comment]
I had 140 repo and all deleted suddenly, what happened

**Reply by ghost** [reply]
That happened to me too but I have a message from gitlocker :/

**Comment by jakob1379** [comment]
any news on this, just figured some of mine have gone missing too....

**Reply by aaakash06** [reply]
Same here. I was working on one and thought of pushing it. What I see is the connection is purged and no repo in the github dashboard.

**Comment by mearskdiaz** [comment]
Same for me, any workaround just to recover the files? Thank you!

**Comment by shettar20** [comment]
This post might help you:
https://devactivity.com/insights/lost-your-github-repository-uncovering-missing-projects-and-boosting-your-github-stats/
